src/modules/dev/http.ts

```
@@ -1,7 +1,22 @@
 import fs from 'node:fs';
 import path from 'node:path';
 import { Hono } from 'hono';
+import { magicLinkTemplate } from '@/shared/services/email/templates/auth/magic-link';
+import { customerPaymentReceipt } from '@/shared/services/email/templates/customer/payment-receipt';
+import { customerPaymentRequest } from '@/shared/services/email/templates/customer/payment-request';
+import { customerPaymentRefundRequest } from '@/shared/services/email/templates/customer/payment-refund-request';
```

ESLint

[error] `@/shared/services/email/templates/customer/payment-refund-request` import should occur before import of `@/shared/services/email/templates/customer/payment-request`

(import/order)